src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -0,0 +1,441 @@
+---
+title: Basic formatting
+description: >
+  Tables display sets of related data across rows and columns. Tables make it easier for users to compare and analyze information.
+---
+
+import BasicFormatting from '../examples/tables/BasicFormatting.tsx';
+import BasicFormattingCode from '!!raw-loader!../examples/tables/BasicFormatting.tsx';
+
+<CodeExample code={BasicFormattingCode}>
+  <BasicFormatting />
+</CodeExample>
+
+## Anatomy
+
+### Parts of a table
+
+![Parts of a table](figma:patterns-tables-basic-formatting-anatomy.png)
+
+1. Columns are vertical groupings of information that categorize and priortize data
+1. Header rows describe the information provided in a column or row
+1. Rows are horizontal groups of data that fall under each column
+
+### Table content
+
+![Wrap and Truncation](figma:patterns-tables-basic-formatting-wrap-and-truncation.png)
+
+1. Header and data cells should be concise
+1. Text wrapping is the default and recommended behavior. Ensure the cells in the row are top aligned to help with
+   scannability.
+1. Keep in mind, truncation in a table can make it harder for users to scan. Use an
+   [ellipsis](/components/typography#ellipsis-text) to truncate long text at the end of the first line and provide a
```

Comment:
Yes since that's the capitalization rule that exists today and we still need to discuss as a team on how we want to move forward in changing this.

cc: @james-sann I've updated our docs

Reply:
@diana-oum Thank you!